otter/models/intents.py

```diff
@@ -78,6 +78,25 @@ def perform_update_group_status(dispatcher, ugs_intent):
 
 
 @attr.s
+class UpdateScalingGroupStatus(object):
+    """
+    Intent to update scaling group status. This is different from
+    :obj:`UpdateGroupStatus` by taking tenant_id and group_id instead
+    of group object
+    """
+    tenant_id = attr.ib()
+    group_id = attr.ib()
+    status = attr.ib()
+
+
+@deferred_performer
+def perform_update_scaling_group_status(log, store, dispatcher, intent):
+    """Perform an :obj:`UpdateScalingGroupStatus`."""
```

Comment:
Hmm. This is not good as lot of code has limited documentation. It leads me to believe that we need much more documentation for effect related code :-|